apps/aecore/test/aecore_sync_SUITE.erl

```diff
@@ -87,46 +85,27 @@ init_per_suite(Config) ->
     Config1.
 
 end_per_suite(Config) ->
-    lists:foreach(
-        fun(Node) ->
-            aecore_suite_utils:stop_node(Node, Config)
-        end,
-        [dev1, dev2, dev3]),
-    ok.
+    stop_devs(Config).
 
 init_per_group(two_nodes, Config) ->
-    config(two_nodes, Config);
+    config({devs, [dev1, dev2]}, Config);
 init_per_group(three_nodes, Config) ->
-    config(three_nodes, Config);
+    config({devs, [dev1, dev2, dev3]}, Config);
 init_per_group(one_blocked, Config) ->
-    Config1 = config(two_nodes, Config),
+    Config1 = config({devs, [dev1, dev2]}, Config),
     preblock_second(Config1),
     Config1;
+init_per_group(faulty_connect, Config) ->
```

Comment:
Is this meant to be deleted as moved to aec_eunit_SUITE?

Reply:
Yes, will remove.